base/file.jl

```diff
@@ -359,14 +359,16 @@ mktemp(parent)
     mktempdir(parent=tempdir())
 
 Create a temporary directory in the `parent` directory and return its path.
+Note that `parent` will not be created; if it doesn't exist an error will be thrown.
```

Comment:
Perhaps simplify to "If `parent` does not exist, [throw an error | throw | error]."?